src/js/disability-benefits/components/AddFilesForm.jsx

```diff
@@ -105,12 +105,18 @@ class AddFilesForm extends React.Component {
           <div key={index} className="document-item-container">
             <Element name={`documentScroll${index}`}/>
             <div className="document-title-size">
-              <div className="document-title-header">
-                <h4 className="title">{file.name}</h4>
+              <div className="document-title-text-container">
+                <div className="document-title-header">
+                  <h4 className="title">{file.name}</h4>
+                </div>
+                <div className="document-size-text">
+                  <p className="size">{displayFileSize(file.size)}</p>
```

Comment:
I just checked and the text-size is exactly the same without the `p` tag, so I can remove this.